.github/workflows/rpk-build.yml

```diff
@@ -60,7 +61,7 @@ jobs:
 
     - name: Upload build artifacts
       uses: actions/upload-artifact@v2
-      if: startsWith(github.ref, 'refs/tags/release-')
+      if: startsWith(github.ref, 'refs/tags/v')  && !contains(github.ref, '-')
```

Comment:
can also hack in regex here too it looks like by setting a context variable conditional on a shell command: https://stackoverflow.com/a/58869470/144213

Reply:
Do you feel that it is a requirement to use a regex here? For now, I have it not publishing anything that does have a dash but that might actually change, as we might want to publish artifacts like `v20.12.2-rc1`. So then a regex might be a bit overkill.

Reply:
not at all. just the sort of bike shedding i enjoy wasting time on